README.rst

```diff
@@ -47,6 +49,14 @@ Getting Started
 
 .. _Running server on device: #browsing-server-on-device
 
+Issues
+---------------
+For questions on development for the App you can post a topic here 
+https://groups.google.com/forum/#!forum/zulip-android
+
+For questions on development for the Server you can post a topic here 
+https://groups.google.com/forum/#!forum/zulip-devel
```

Comment:
Cool!  A few suggestions:
(1) I'd probably be a bit verbose and say e.g. "development for the Zulip Android app" and "for the Zulip server".

(2) Should we also point people at the Android stream on zulip.tabbott.bet?

(3) Do we also want to suggest GitHub issues for bug reports?  Maybe a clearer approach would be tor rename this section "Getting help".